### PR TITLE
add some better dev setup tutorial to make you up to speed fast

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: docs/tutorials/development.md [hunk 1]
```diff
@@ -1,3 +1,47 @@
+## Local Setup
+
+### Build Skipper Binary
+
+Clone repository and compile with [Go](https://golang.org/dl).
+
+```sh
+% git clone https://github.com/zalando/skipper.git
+% cd skipper
+% make skipper
```
Comment (AlexanderYastrebov) [inline docs/tutorials/development.md:10]: Shell prompt makes it a bit harder to copy paste the command. Also below there is no shell prompt. Should we remove it maybe?

File: docs/tutorials/development.md [hunk 2]
```diff
@@ -1,3 +1,47 @@
...
+```
+
+binary will be ./bin/skipper
```
Comment (AlexanderYastrebov) [inline docs/tutorials/development.md:13]: Maybe quote with backticks like `./bin/skipper`?

File: docs/tutorials/development.md [hunk 3]
```diff
@@ -1,3 +1,47 @@
...
+
+### Run Skipper as Proxy with 2 backends
+
+As a small example, we show how you can run one proxy skipper and 2
+backend skippers.
+
+Start the proxy that listens on port 9999 and serves all requests with a single route, that
+proxies to two backends with the round robin algorithm:
```
Comment (AlexanderYastrebov) [inline docs/tutorials/development.md:21]: with the round robin -> using the round robin

File: docs/tutorials/development.md [hunk 4]
```diff
@@ -1,3 +1,47 @@
...
+```
+./bin/skipper -inline-routes='r1: * -> <roundRobin, "http://127.0.0.1:9001", "http://127.0.0.1:9002">' --address :9999
+```
+
+Start two backends, with similar routes, one response with "1" and the
+other with "2" in the HTTP response body:
```
Comment (AlexanderYastrebov) [inline docs/tutorials/development.md:27]: response -> responds

**Comment by AlexanderYastrebov** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**